src/pages/Member/SignUpForm/SignUpForm.js

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from 'react';
+import './SignUpForm.scss';
+
+const SignUpForm = () => {
+  const [inputState, setInputState] = useState({
+    email: '',
+    password: '',
+    name: '',
+    address: '',
+    phoneNumber: '',
+  });
+
+  const [radioGroup, setRadioGroup] = useState({
+    isFamily: false,
+    isNotFamily: false,
+  });
+
+  const changeRadio = event => {
+    let obj = {};
```

Comment:
```suggestion
    const obj = {
       [value]:checked
    };
```
선언, 프로퍼티 추가 나눠서 하지말고 한번에 가능해보이네요!